src/LibraryViewExtensionMSWebBrowser/LibraryViewExtensionMSWebBrowser.csproj

```diff
@@ -9,6 +9,12 @@
     <RootNamespace>Dynamo.LibraryViewExtensionMSWebBrowser</RootNamespace>
     <AssemblyName>LibraryViewExtensionMSWebBrowser</AssemblyName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputPath>bin\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <OutputPath>bin\Release\</OutputPath>
+  </PropertyGroup>
```

Comment:
@QilongTang I did similar changes like the ones you did for switch from cefSharp to WebBrowser component, I think is you don't specify an output path it will take the one from "$(SolutionDir)Config\CS_SDK.props" (it will be generating the dll files in Dynamo\bin\AnyCPU\Debug).
Please check the changes done for switching to WebBrowser:
https://github.com/DynamoDS/Dynamo/pull/11957/files